lineage/tests/test_StateEstimation.py

```diff
@@ -0,0 +1,81 @@
+""" Unit test file. """
```

Comment:
New unit test file with one state lineages/fitting and evaluation of parameter estimation.